PIL/ImageGrab.py

```diff
@@ -72,7 +72,6 @@ def grabclipboard():
         os.unlink(filepath)
         return im
     else:
-        debug = 0  # temporary interface
         data = Image.core.grabclipboard(debug)
```

Comment:
It's not clear for me. `debug` var is never defined but is still used there. Possible should be `Image.core.grabclipboard(0)`.

Reply:
Thanks for catching that! I've removed this change from here, and created PR #2708